app/src/main/java/com/dozingcatsoftware/bouncy/BouncyActivity.java

```diff
@@ -241,10 +241,8 @@ void showPausedButtons() {
     @Override protected void onActivityResult(int requestCode, int resultCode, Intent intent) {
         super.onActivityResult(requestCode, resultCode, intent);
 
-        switch (requestCode) {
-            case ACTIVITY_PREFERENCES:
-                updateFromPreferences();
-                break;
+        if (requestCode == ACTIVITY_PREFERENCES) {
```

Comment:
I prefer to keep the switch when dealing with codes like this, even if there's only one at the moment.

Reply:
Mm, okay.